src/components/TxBuilderAttributes.js

```diff
@@ -36,7 +40,7 @@ export default function TxBuilderAttributes(props) {
           value={attributes['fee']}
           onUpdate={(value) => {onUpdate('fee', value)}}
           />
-        <p className="optionsTable__pair__content__note">The <a href="https://www.stellar.org/developers/guides/concepts/fees.html">network base fee</a> is currently set to 100 stroops (0.00001 lumens). Transaction fee is equal to base fee times number of operations in this transaction.</p>
+        <p className="optionsTable__pair__content__note">The <a href="https://www.stellar.org/developers/guides/concepts/fees.html">network base fee</a> is currently set to {attributes['fee']} stroops ({attributes['fee'] / 1e7} lumens). Transaction fee is equal to base fee times number of operations in this transaction.</p>
```

Comment:
"Base fee" is a Stellar network term, I believe this should stay unchanged https://developers.stellar.org/docs/glossary/fees/#base-fee
```suggestion
        <p className="optionsTable__pair__content__note">The <a href="https://www.stellar.org/developers/guides/concepts/fees.html">network base fee</a> is currently set to 100 stroops (0.00001 lumens). Transaction fee is equal to base fee times number of operations in this transaction.</p>
```

Reply:
My understanding is that this is now (starting on Protocol 13) dynamic and can be voted, so we can't rely on a fixed number for the base fee anymore. Does that sounds right @ire-and-curses?

Reply:
Yeah this is correct. In fact it could always be changed through a vote, but that couldn't happen before without potentially invalidating pre-signed transactions. Now those can be fee bumped, so in practical terms it can change now (although such changes will be infrequent).

Reply:
Thanks for the clarity. Keeping the change then based on above comments @vcarl

Reply:
Ah I see, yeah that makes sense.